RUFAS/input_manager.py

```diff
@@ -726,6 +727,33 @@ def get_data_keys_by_properties(self, target_properties: str) -> list[str]:
 
         return data_keys
 
+    def delete_data(self, data_address: str) -> bool:
+        info_map = {
+            "class": self.__class__.__name__,
+            "function": self.delete_data.__name__,
+        }
+        keys = data_address.split(".")
+
+        try:
+            data_parent = self.data_validator.extract_value_by_key_list(self.__pool, keys[:-1])
+            removed_value = data_parent.pop(keys[-1])
+            ts = Utility.get_timestamp(include_millis=True)
+            self.__delete_data_logs_pool[ts] = f"InputManager.delete_data() called for {keys}"
```

Comment:
Since we're logging the error if it fails, maybe log the successful path completing as well.